src/client/components/Footer/Footer.css

```diff
@@ -0,0 +1,109 @@
+body {
+    margin:0;
+    padding:0;
+    height:100%;       
+   }  
+
+.footer {  
+    position: absolute;        
+    height: 105px;
+    width: 100%;
+    left:0px;   
+    background-color:#ffffff;
+    box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.5);  
+    bottom: 0;
+    min-width:1100px;    
+  }
+
+  .footer p {
+    position:absolute;
+    height: 21px;
```

Comment:
Have you checked this on mobile? Its always a bit dangerous to give elements fixed height as text changes when the screen gets smaller